### DISCUSSION TITLE
Flight controller choice

### DISCUSSION THREAD

**Discussion by This-guy-breaks-shit** [iNavFlight/inav#6614] [Q&A]
As far as I've seen, using [this](https://airbot.uk/product/omninxt-f7-flight-controller/) flight controller would be a suitable choice for this firmware. I'd just like to confirm that this is a good choice before buying. Any thoughts?

**Comment by DzikuVx** [comment]
Please take a look here https://github.com/iNavFlight/inav/wiki/Welcome-to-INAV,-useful-links-and-products

**Reply by This-guy-breaks-shit** [reply]
already have. Just not quite sure how important it is to use the recommended and if I'd be missing out on anything if I didn't. The flight controller I found looks more appealing as it has secondary vibration reduction on the pair of IMUs, along with having a pair of IMUs

**Reply by avsaase** [reply]
As long as the fc you choose has an INAV target you should be fine.

**Reply by This-guy-breaks-shit** [reply]
It doesn't mention iNav on the website on that particular model but I'll inquire further. If not then I'll go for one of the recommended choices.

**Reply by avsaase** [reply]
You can view the supported targets in the Configurator or here under assets: https://github.com/iNavFlight/inav/releases/tag/2.6.0

**Reply by This-guy-breaks-shit** [reply]
I've already seen that, and I'm looking into them. I'll probably get one of them anyway
